test/library/standard/BigInteger/apiTest.chpl

```diff
@@ -0,0 +1,221 @@
+use BigInteger;
+
+config const executeLocale = 0;
+
+const a = new bigint("123456789012345678901234567890");
+const b = new bigint("123456789012345678901234567891");
+const c = new bigint("246913578024691357802469135780");
+const d = new bigint("-246913578024691357802469135780");
+
+on Locales[min(Locales.domain.high, executeLocale)] {
+  assert((a+1) == b);
+  assert((a+(1:bigint)) == b);
+  assert(b == (a+1));
+  assert(b != a);
+  assert(b > a);
+  assert(b >= a);
+  assert(!(b < a));
+  assert(!(b <= a));
+
+  assert((a*2) == c);
+  assert((c-a) == a);
+  assert(c == (a + a));
+  assert((c+1) == (a+b));
+  assert(d == -c);
+
+  forall i in -10..10 do
+    for j in -10..10 do
+      if j != 0 {
+        assert(div(i:bigint, j:bigint) == div(i,j));
+        assert(mod(i:bigint, j:bigint) == mod(i,j));
+        assert(mul(i:bigint, j:bigint) == mul(i,j));
+      }
+
+  proc div(a, b) {
+    return a/b;
+  }
+
+  proc mod(a, b) {
+    return a%b;
+  }
+
+  proc mul(a, b) {
+    return a*b;
+  }
```

Comment:
What's the motivation for these wrappers? Just easier transliteration?

Reply:
Ya, I guess it seemed like _possibly_ a different case since it was using a `proc`, but it was mainly just a transliteration.

Would you prefer just putting those inline without a `proc`?

Reply:
Mild preference